### PR TITLE
add a simple example of reading TextGrid file to README.md

### PR DESCRIPTION
**Pull request by Miopas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,81 @@ Synopsis:
 ---------
 
 See the docstrings in `textgrid.py`
+
+Example:
+---------
+This is a simple example of reading a TextGrid file. 
+```
```
Comment (kylebgorman) [inline README.md:32]: Blank line, then use "```python" so the Markdown renderer can use syntax highlighting.

File: README.md [hunk 2]
```diff
@@ -25,3 +25,81 @@ Synopsis:
...
+import textgrid
+
+# Read TextGrid object from file
+tg = textgrid.TextGrid.fromFile('test.TextGrid')
+
+# Interval example
+print ("------- Interval example -------")
```
Comment (kylebgorman) [inline README.md:39]: Please use your best Python style---this is supposed to be a readable example for newbies, right?

* Please don't use space between "print" and the parentheses, I didn't even know the Python grammar allowed that and it's very distracting.
* Comments should have the syntax of a sentence and be punctuated as such.

File: README.md [hunk 3]
```diff
@@ -25,3 +25,81 @@ Synopsis:
...
+print (tg[0]) # item[1]
```
Comment (kylebgorman) [inline README.md:40]: While I would personally omit these comments altogether (I don't find them helpful) if you want to keep them them should match the syntax of the TextGrid file...e.g., `item [1]` not `item[1]`.

File: README.md [hunk 4]
```diff
@@ -25,3 +25,81 @@ Synopsis:
...
+        points: size = 2
+        points [1]:
+            number = 0.25
+            mark = """event"""
+        points [2]:
+            number = 0.75
+            mark = """event"" with quotes again"
+```
+
+By runing the code above, the output is:
```
Comment (kylebgorman) [inline README.md:92]: "running" not "runing". But just say "the following is the output of the above snippet:" or something like that.

**Comment by Miopas** [comment]
Thanks for your suggestions! I have revised what you mentioned. Please feel free to let me know if any further revision is needed.

**Comment by kylebgorman** [comment]
LGTM.

**Pull request merged**